smart_news_query_embeddings/tests/test_specificity_scores.py

```diff
@@ -0,0 +1,93 @@
+"""
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+import pandas as pd
+import numpy as np
+from smart_news_query_embeddings.preprocessing.specificity_scores import \
+get_specificity_scores, get_token_scores, get_normalized_scores
+
+class TestSpecificityScores(unittest.TestCase):
+
+	def test_get_specificity_scores(self):
+
+		"""
+		Test that we get back as many scores as we put in sentences.
+		"""
+
+		sentences = [
+			"This is sentence 1 about Wall Street",
+			"This is sentence 2 about Donald Trump",
+			"This is sentence 3 about New York City",
+			"Tesla announces first ever car that can drive in space",
+			"A look at last night's game between the Warriors and the Lakers, sponsored by Tesla"
+		]
+		scores = get_specificity_scores(sentences)
+		self.assertEqual(scores.shape[0], len(sentences))
```

Comment:
does this function require a call to an external service? If you can you refactor the function to 
1. get_responses(sentences)
2. get_scores_from_responses(responses)

Then you only need to test get_scores_from_responses since the first one should be taken care of by the api developer